scan/spec/test_command_generator_spec.rb

```diff
@@ -5,6 +5,31 @@
     @project = FastlaneCore::Project.new(config)
   end
   before(:each) do
+    @valid_simulators = "== Devices ==
+-- iOS 9.3 --
+    iPhone 4s (238767C4-AF29-4485-878C-7011B98DCB87) (Shutdown)
```

Comment:
Just wondering why Xcode 6 support is needed? What's the harm in saying Xcode 7+ support or if you want 6 support, use an older version of Fastlane. Just trying to get a feel for the Fastlane mindset :)